src/utils/index.ts

```diff
@@ -163,23 +163,25 @@ export const isGifMessage = (message: UserMessage | FileMessage): boolean => (
 );
 export const isAudioMessage = (message: FileMessage): boolean => message && isFileMessage(message) && isAudio(message.type);
 export const isAudioMessageMimeType = (type: string): boolean => (/^audio\//.test(type));
+export const isVoiceMessageMimeType = (type: string): boolean => (/^voice\//.test(type));
 export const isVoiceMessage = (message: Nullable<FileMessage | UserMessage>): boolean => {
   // ex) audio/m4a OR audio/m4a;sbu_type=voice
-  if (!(message && isFileMessage(message))) {
+  if (!(message && isFileMessage(message)) || !(message as FileMessage).type) {
     return false;
   }
-  const [mimeType, typeParameter] = message.type.split(';');
-  if (!isAudioMessageMimeType(mimeType) || !typeParameter) {
+  const [mimeType, typeParameter] = (message as FileMessage).type.split(';');
+
+  if (!isAudioMessageMimeType(mimeType)) {
     return false;
   }
-  const [key, value] = typeParameter.split('=');
-  if (key === 'sbu_type' && value === 'voice') {
-    return true;
+
+  if (typeParameter) {
```

Comment:
Do we want to pass the case `typeParameter === undefined` to [the last return statement](https://github.com/sendbird/sendbird-uikit-react/pull/643/files#diff-6147e3c1761df71fd40952dbcbac388a45f80b8c318f367ef41048351a2c0c99R183)?

Reply:
Yes I do